packages/pn-commons/src/components/__test__/CustomDatePicker.test.tsx

```diff
@@ -51,11 +60,15 @@ describe('test CustomDatePicker languages', () => {
   ];
 
   it.each(languages)('check january month to be $month in $language', (language) => {
-    const { container, getByRole } = render(<RenderDatePicker language={language.language} />);
+    const { container, getByLabelText } = render(<RenderDatePicker language={language.language} />);
```

Comment:
why the use of getByLabelText instead of getByRole?

Reply:
the previous test failure because not found role

Reply:
maybe fails because by default it renders mobile view and the mobile view doesn't have the button. revert the changes and force the desktop view using createMatchMedia

Reply:
how do i emulate without using the specific hook?